tiledb/api/c_api/vfs/test/unit_capi_vfs.cc

```diff
@@ -558,6 +558,7 @@ TEST_CASE("C API: tiledb_vfs_open argument validation", "[capi][vfs]") {
 TEST_CASE("C API: tiledb_vfs_write argument validation", "[capi][vfs]") {
   ordinary_vfs_fh x;
   const void* buffer = nullptr;
+  const void* non_null_buffer = (void*)0x1;
```

Comment:
For better portability, it would be better to return an address to some stack-allocated buffer to ensure it's a real address. There have been platforms where all "small" pointer values were not valid.

Reply:
Well this is just exists so that the size is rejected. But giving it a valid buffer won't hurt anything so I'll go ahead and update it.

Reply:
Done.